### ISSUE TITLE
feature request: additional factors on X axis

### ISSUE THREAD

**Issue by mneinast** [MRMacArthur/TracebaseView#3]
you might already be doing this, but it would be nice to select additional variables to separate data on the X axis.  For example, the x axis variable could be set to `interact(var1, var2)`.  Perhaps limiting this to only one additional variable on X would keep things clean.

**Comment by MRMacArthur** [comment]
Definitely, good point. I was planning to add a color fill option, which would give the ability to split the x-axis grouping by one additional variable. Based on the data, do you think there are enough variables that we would want to have the ability to split the x-axis grouping by more than one additional variable?

**Comment by mneinast** [comment]
Some stacked bar plots plots in the 2020 fluxomics paper used color to
indicate tracer and had two variables along x axis (ie diet, feeding status)

**Comment by MRMacArthur** [comment]
Added temporary fix, optional fill color parameter which allows one additional split on the x-axis. Need to think more about the best/most flexible way to address this.
